blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/FavoritesStatePreloader.java

```diff
@@ -56,7 +57,7 @@ protected FetchData getFetchData(@Nonnull BlueUrlTokenizer blueUrl) {
         User jenkinsUser = User.current();
 
         if (jenkinsUser != null) {
-            UserImpl blueUser = new UserImpl(jenkinsUser);
+            UserImpl blueUser = new UserImpl(OrganizationUtil.getOrganization(OrganizationUtil.getOrganizationNameFromURL(), true), jenkinsUser);
```

Comment:
Right - because the permissions are at the folder (org) level. That makes sense.

Could `getOrganizationNameFromURL` be moved to `OrganizationImpl` and be marked as `@Restricted`? 

Then delete OrganizationUtils completely